.github/workflows/pr-issue-validator.yaml

```diff
@@ -53,6 +53,15 @@ jobs:
         
         ### For ex: Fixes devtron-labs/devtron#2123
         pattern3="((Fixes|Resolves) devtron-labs/devtron#[0-9]+)"
+
+        ### For ex: Fixes [#4839](https://github.com/devtron-labs/devtron/issues/4839)
+        pattern4="(Fixes|Resolves):?\s+\[#([0-9]+)\]"
+
+        ### For ex: Fixes devtron-labs/devops-sprint#2123
```

Comment:
Use a pattern that either matches both 
https://github.com/devtron-labs/devtron/issues/2123 
and 
#devtron-labs/devops-sprint#2123